pio/lib/Sender/Sender.h

```diff
@@ -26,6 +26,7 @@ class SenderClass
   bool sendGenericPost(String server, String uri, uint16_t port = 80);
   bool sendHTTPSPost(String server, String uri);
   bool sendInfluxDB(String server, uint16_t port, String db, String name, String username, String password);
+  bool sendInfluxDBHTTPS(String server, uint16_t port, String db, String name, String username, String password);
```

Comment:
bool sendInfluxDB(String server, uint16_t port, String db, String name, String username, String password, **bool bUseHTTPS**); ?

Reply:
Reimplemented to use a bool/checkbox, as suggested.